### ISSUE TITLE
add CRON travis job

### ISSUE THREAD

**Issue by AdamSpannbauer** [bmewing/requirements#49]
Once CI stuff is merged into master, a CRON build should be set-up in Travis so we can be alerted about breaking changes in dependencies/R-devel.

**Comment by bmewing** [comment]
I set this up in Travis-CI today to run monthly unless there's been a build in the last 24 hours
